docs/admin.md

```diff
@@ -0,0 +1,59 @@
+## Admin Manual for Definitely Typed
+
+Welcome! This is a resource for the person who is on call for DefinitelyTyped. The TypeScript team always has someone 
+assigned to DefinitelyTyped duty, where they do a week on-call. You can see the [schedule here](http://aka.ms/DTRotation).
+
+When on-call, your goal is to try keep on top of the many open PRs for DefinitelyTyped, they are categorized into 
+different sections inside the [Projects board](https://github.com/DefinitelyTyped/DefinitelyTyped/projects/4) on this repo.
+
+You should scan from left to right through the boards, and ideally try to start at the latest PR and work your way 
```

Comment:
When you say latest PR; Do you mean oldest PR?

Reply:
Thanks!